diff --git a/node_modules/@argos-ci/playwright/dist/index.mjs b/node_modules/@argos-ci/playwright/dist/index.mjs
index dd8892c..3f0d600 100644
--- a/node_modules/@argos-ci/playwright/dist/index.mjs
+++ b/node_modules/@argos-ci/playwright/dist/index.mjs
@@ -28,7 +28,9 @@ function waitForFontLoading() {
 }
 // Check if the images are loaded
 function waitForImagesLoading() {
-    return Array.from(document.images).every((img)=>img.complete);
+    const allImages = Array.from(document.images);
+    allImages.forEach(img => img.loading = "eager")
+    return allImages.every((img)=>img.complete);
 }
 async function argosScreenshot(page, name, { element , has , hasText , ...options } = {}) {
     if (!page) throw new Error("A Playwright `page` object is required.");
